### DISCUSSION TITLE
couldn't send metrics data to skywalking OAP server

### DISCUSSION THREAD

**Discussion by veejayvb** [apache/skywalking#11634] [Q&A]
i have a a demo react application and i deployed skywalking oap server ui and postgres in gke cluster , while deploying my react application in cluster its sending send data to  default configuration which is 
http://10.4.20.69:12800/core/default/restPort/browser/perfData
but my skywalking clusterIP is 10.4.20.69 
i tried with following congiuration 

import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
@feature: nodejs-agent-frontend;
import ClientMonitor from 'skywalking-client-js';

try {
  ClientMonitor.register({
    collector: 'http://10.4.20.69:12800/core/default/restPort', // SkyWalking OAP server address
    service: 'test-ui', // Your project ID
    pagePath: '/', // 
    serviceVersion: 'v1.0.0', 
    autoTracePerf: true,
    // enableSPA: true, // Not monitoring page hashchange events (for Single Page Apps)
    traceTimeInterval: 20000, 
  });
} catch (error) {
  console.error('Error setting up monitoring:', error);
}

ReactDOM.render(
  <App />,
  document.getElementById('root')
);
![Screenshot from 2023-12-07 11-19-22](https://github.com/apache/skywalking/assets/109889109/d2807635-0606-485e-bca2-9e76d517c92b)
can anyone help me with this sending metrics data to skywalking oap server

**Comment by wu-sheng** [comment]
Where is the prefix of `/core/default/restPort/browser/perfData` from?

The doc listed the address and URI explicitly, no extra root path. https://skywalking.apache.org/docs/main/next/en/api/browser-http-api-protocol/#post-httplocalhost12800browserperfdata

**Reply by veejayvb** [reply]
initally i deployed skywalking components in cluster 
![deployed](https://github.com/apache/skywalking/assets/109889109/20b15617-0bed-4c7a-a8e5-b213461736a4)

after i deployed nodejs application with instrumentation code 
![nodeinstr](https://github.com/apache/skywalking/assets/109889109/5eea78ea-a457-49d9-b5d5-56bd6436947b)

i were able to send metrics to skywalking oap as result i could see it's data in skywalking UI 
![ui](https://github.com/apache/skywalking/assets/109889109/d3b08e87-19c1-4eff-b826-f770ba9dfd61)
now i trying to connect  my react application to skywalking oap server using 
![reactinstr](https://github.com/apache/skywalking/assets/109889109/12b0a3ba-f5df-4c1c-90c6-97f8ebe80c7b)
i couldn't send any data to skywalking oap server 
what needs to be done for react application to see same results as nodejs getting 
both are sending metrics to same clusterIP

**Reply by wu-sheng** [reply]
I mean `/core/default`, this doesn't exist in our code base, where/how get it added?

**Reply by veejayvb** [reply]
https://github.com/apache/skywalking-client-js                           
you could see it here in README file in this package 
// Report collected data to `http:// + window.location.host + /browser/perfData` in default
ClientMonitor.register({
  # Use core/default/restPort in the OAP server.
  # If External Communication Channels are activated, `receiver-sharing-server/default/restPort`,
  # ref to https://skywalking.apache.org/docs/main/latest/en/setup/backend/backend-expose/
  collector: 'http://127.0.0.1:12800', 
  service: 'test-ui',
  pagePath: '/current/page/name',
  serviceVersion: 'v1.0.0',
});

**Reply by wu-sheng** [reply]
But we only use `host` + `/browser/perfData`. You are not.
